### DISCUSSION TITLE
Bootstrap RTL components

### DISCUSSION THREAD

**Discussion by gnasim** [twbs/bootstrap#32623] [Q&A] [answered]
How I can use some components of Bootstrap RTL not all the bootstrap.
So when compile scss file, I need to compile som component as buttons,.. etc not all

**Comment by ffoodd** [comment, marked as answer]
RTL dist CSS files are post-processed from LTR dist CSS files, meaning that you first need to make your custom build (dropping components) from Sass sources for LTR, then run RTLCSS by yourself to output the RTL files.

[This is explained in our docs](https://getbootstrap.com/docs/5.0/getting-started/rtl/#customize-from-source) —but might probably be made clearer— and mostly depends on [Sass customization](https://getbootstrap.com/docs/5.0/customize/sass/). Feel free to open an issue or a PR if you have any improvement to suggest :)
